### DISCUSSION TITLE
ARM docker image of jdk 11

### DISCUSSION THREAD

**Discussion by pasmimmo** [microsoft/openjdk#321] [Q&A] [answered]
Hello to all, 
I'm here to ask if there is a docker image of Microsoft jdk 11 that support arm and multi-arch, or if there is a release plan.

Maybe is my mistake but in official repo i cannot find anything related to arm v8 https://mcr.microsoft.com/product/openjdk/jdk/tags
Thanks for help

**Comment by karianna** [comment]
Hi @pasmimmo - I'm afraid there are no plans to support either of those platforms.

**Reply by pasmimmo** [reply]
Thank you for your answer @karianna,
Targeting Azure arm machines, are there any suggestions on non Microsoft alternatives for jdk? 
I had thought of using Eclipse Temurin, any feedback on this?

**Reply by karianna** [reply]
Apologies - re-reading this properly! The Linux ARM container image support is currently under discussion but I doubt we will be supporting multi-arch.  If you have an immediate need then the Eclipse Temurin images from DockerHub are your next best bet yes.

**Reply by pasmimmo** [reply]
Thank you.

**Comment by brunoborges** [comment, marked as answer]
Please follow this issue: https://github.com/microsoft/openjdk-docker/issues/46
